docs/source/install.rst

```diff
@@ -0,0 +1,71 @@
+Installation
+============
+
+Project Koku is divided into logical components. To achieve a fully functional
+application, it requires the deployment of multiple micro-applications.
+
+The primary applications for Project Koku are:
+
+* Koku API (Reporting and Query API service)
+* Koku UI (Front-end Web UI)
+* Masu (Data ingestion service)
+
+For development and testing, Project Koku also includes:
+
+* Nise (Test data generator)
+* Hansei (Test automation framework)
```

Comment:
You can check with @LandoCalrizzian, but this repo may be going away as they are moving automation to the insights gitlab.